swarm/api/client/client.go

```diff
@@ -489,6 +502,17 @@ func (c *Client) TarUpload(hash string, uploader Uploader, defaultPath string, t
 	if err != nil {
 		return "", err
 	}
+
+	opentracing.GlobalTracer().Inject(
```

Comment:
@justelad I don't like this, because users might also use this client - we don't want `opentracing` headers to be added always.

Reply:
I suggest we get rid of this `opentracing` from here, so that we can merge this PR as it is, because it is getting large.